definition/rest/v1/settings.ts

```diff
@@ -10,38 +10,38 @@ type SettingsUpdatePropsActions = {
 export type OauthCustomConfiguration = {
 	_id: string;
 	clientId?: string;
-	custom: unknown;
+	custom: boolean;
 	service?: string;
-	serverURL: unknown;
-	tokenPath: unknown;
-	identityPath: unknown;
-	authorizePath: unknown;
-	scope: unknown;
+	serverURL: string;
+	tokenPath: string;
+	identityPath: string;
+	authorizePath: string;
+	scope: string;
 	loginStyle: 'popup' | 'redirect';
-	tokenSentVia: unknown;
-	identityTokenSentVia: unknown;
-	keyField: unknown;
-	usernameField: unknown;
-	emailField: unknown;
-	nameField: unknown;
-	avatarField: unknown;
-	rolesClaim: unknown;
-	groupsClaim: unknown;
-	mapChannels: unknown;
-	channelsMap: unknown;
-	channelsAdmin: unknown;
-	mergeUsers: unknown;
-	mergeRoles: unknown;
-	accessTokenParam: unknown;
-	showButton: unknown;
-
-	appId: unknown;
+	tokenSentVia: string | boolean;
+	identityTokenSentVia: string | boolean;
+	keyField: string | boolean;
+	usernameField: string | boolean;
+	emailField: string | boolean;
+	nameField: string | boolean;
+	avatarField: string | boolean;
+	rolesClaim: string;
+	groupsClaim: string;
+	mapChannels: string;
+	channelsMap: string;
+	channelsAdmin: string;
+	mergeUsers: boolean;
+	mergeRoles: boolean;
+	accessTokenParam: string;
+	showButton: boolean;
+
+	appId: string;
 	consumerKey?: string;
 
-	clientConfig: unknown;
-	buttonLabelText: unknown;
-	buttonLabelColor: unknown;
-	buttonColor: unknown;
+	clientConfig: { provider: SettingValue };
+	buttonLabelText: SettingValue;
+	buttonLabelColor: SettingValue;
+	buttonColor: SettingValue;
```

Comment:
what's the reason to use `SettingValue`? looks like they're just `string`

Reply:
On the `app/meteor-accounts-saml/server/lib/settings.ts`, there's a `settings.get` that returns a SettingValue, so I wasn't sure if this could affect something. But I'll change it to string! Thanks!